RPGManager_development/src/rpg_database/character_sheet/Background.java

```diff
@@ -69,6 +88,7 @@ public void setSelfInSheet(CharacterSheet characterSheet) {
 					Fields.BASECLASS).toString()));
 		}
 		characterSheet.characterData.put(Fields.BACKGROUND, this);
+		characterSheet.setData(Fields.LANGUAGES, this.language);

```

Comment:
Ez direkt van így, ami hasonló elvű program volt annál én ezt szoktam meg, hogy ha base classot, backgroundot állítasz akkor resetel minden eddig beállított dolgot. Viszont átállítani át lehet a defaultokat ezután.